nextstrain/cli/runner/conda.py

```diff
@@ -610,9 +605,10 @@ def package_distribution(channel: str, package: str, version: str = None, label:
 
     if (system, machine) == ("Linux", "x86_64"):
         subdir = "linux-64"
-    elif (system, machine) in {("Darwin", "x86_64"), ("Darwin", "arm64")}:
-        # Use the x86 arch even on arm (https://docs.nextstrain.org/en/latest/reference/faq.html#why-intel-miniconda-installer-on-apple-silicon)
```

Comment:
Can this doc page go away too? (Later, non-blocking)

Reply:
https://github.com/nextstrain/docs.nextstrain.org/pull/246